tests/spread/pack/organize-overlay/imagecraft.yaml

```diff
@@ -0,0 +1,95 @@
+name: organize-to-overlay
+version: "0.1"
+base: bare
+build-base: ubuntu@24.04
+
+platforms:
+  amd64:
+
+volumes:
+  pc:
+    schema: gpt
+    structure:
+      - name: efi
+        type: C12A7328-F81F-11D2-BA4B-00A0C93EC93B
+        filesystem: vfat
+        role: system-boot
+        filesystem-label: UEFI
+        size: 256M
+      - name: rootfs
+        type: 0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        filesystem: ext4
+        filesystem-label: writable
+        role: system-data
+        size: 5G
+
+filesystems:
+  default:
+    - mount: /
+      device: (volume/pc/rootfs)
+    - mount: /boot/efi/
+      device: (volume/pc/efi)
+
+parts:
+  # bootstrap a simple rootfs
+  rootfs:
+    plugin: nil
+    build-packages: ["mmdebstrap"]
+    override-build: |
+      mmdebstrap --arch $CRAFT_ARCH_BUILD_FOR \
+       --mode=sudo \
+       --format=dir \
+       --variant=minbase \
+       --include=apt \
+       noble \
+       $CRAFT_PART_INSTALL/ \
+       http://archive.ubuntu.com/ubuntu/
+      # mmdebstrap filed /dev with special files. Clean it.
+      rm -r $CRAFT_PART_INSTALL/dev/*
+      # Prepare needed directory for EFI
+      mkdir $CRAFT_PART_INSTALL/boot/efi
+    organize:
+      "*": (overlay)/
+
+  # add packages using chroot and apt to have maintainer scripts run
+  packages:
+    plugin: nil
+    overlay-script: |
```

Comment:
how come we don't use `overlay-packages` here, since there is a part organizing to the overlay?